packages/http/src/mocker/index.ts

```diff
@@ -169,15 +170,41 @@ export function createInvalidInputResponse(
           if (error instanceof ProblemJsonError && error.status === 404) {
             return error;
           }
-          return securityValidation
-            ? createUnauthorisedResponse(securityValidation.tags)
-            : createUnprocessableEntityResponse(failedValidations);
+          return createResponseForViolations(failedValidations);
         })
       )
     )
   );
 }
 
+function getExpectedCodesForViolations(failedValidations: NonEmptyArray<IPrismDiagnostic>): NonEmptyArray<number> {
```

Comment:
```
const findValidationByCode = (validations: NonEmptyArray<IPrismDiagnostic>, code: number) =>
  validations.find(validation => validation.code === code);
```
feels like we can add a function and use it in 4 places ;)

Reply:
Fixed